### PR TITLE
docs: Make space optional in scope regex (#661)

### PR DESCRIPTION
**Pull request by pnicolcev-tulipretail**
Signed-off-by: Philip Nicolcev <phil.nicolcev@tulip.io>

### CODE REVIEW

**Comment by pnicolcev-tulipretail** [comment]
Running genswag caught another change in there. Can remove if that's a problem.

File: docs/api.swagger.json [hunk 1]
```diff
@@ -2254,12 +2254,12 @@
           "x-go-name": "Public"
         },
         "redirect_uris": {
-          "description": "RedirectURIs is an array of allowed redirect urls for the client, for example: http://mydomain/oauth/callback .",
```
Comment (arekkas) [inline docs/api.swagger.json]: This is weird, why is the description gone?

Comment (pnicolcev-tulipretail) [reply docs/api.swagger.json]: I don't know, i assumed you'd removed it. Strange that's the only thing that changed. I tried running genswag again, same result. I'll investigate what I did wrong.

Comment (arekkas) [reply docs/api.swagger.json]: Hm strange, maybe a version mismatch in go-swagger. The corresponding line is here: https://github.com/ory/hydra/blob/master/client/client.go#L39-L40

I think the issue is `, for example: http://mydomain/oauth/callback .` which is apparently parsed as an example section. You could try rewriting it to `, for example http://mydomain/oauth/callback`

Comment (arekkas) [reply docs/api.swagger.json]: Damn, I was just a bit late! :D

Comment (pnicolcev-tulipretail) [reply docs/api.swagger.json]: @arekkas fixed

**Comment by arekkas** [comment]
Thank you!

**Pull request merged**